### PR TITLE
Added save toml file func

### PR DESCRIPTION
**Pull request by ssd04**

### CODE REVIEW

File: core/file_test.go [hunk 1]
```diff
@@ -69,6 +70,62 @@ func TestLoadTomlFile_FileExitsShouldPass(t *testing.T) {
...
+		t.Parallel()
+
+		cfg := &TestStruct{}
+
+		dir := t.TempDir()
+		fileName := dir + "/testFile1"
+
+		err := core.SaveTomlFile(cfg, fileName)
+		if _, errF := os.Stat(fileName); errF == nil {
+			_ = os.Remove(fileName)
```
Comment (iulianpascalau) [inline core/file_test.go:108]: this is not required as when the test will end, the whole t.TempDir() will be deleted

File: core/file_test.go [hunk 2]
```diff
@@ -69,6 +70,62 @@ func TestLoadTomlFile_FileExitsShouldPass(t *testing.T) {
...
+		}
+
+		assert.Nil(t, err)
+	})
+}
+
```
Comment (iulianpascalau) [inline core/file_test.go:128]: maybe add an "integration test" between the SaveTomlFile and LoadTomlFile? Create a mock struct, add some fields, save it, load it, and compare expected vs loaded structs. Should not find errors in the process.

File: core/file_test.go (hunk 1 above)
Comment (ssd04) [reply core/file_test.go]: removed

File: core/file_test.go (hunk 2 above)
Comment (ssd04) [reply core/file_test.go:127]: updated test to include save and load

**Review by BeniaminDrasovean — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**